### DISCUSSION TITLE
Will TensorRT-LLM be available within Triton or will it be a separate server ?

### DISCUSSION THREAD

**Discussion by MatthieuToulemont** [triton-inference-server/server#6290] [Q&A] [answered]
Like many, I am pretty stoked about the TensorRT-LLM announcement. 

I am wondering if this will be accessible from within Triton as a specific backend or will we need to run a separate process to benefit from TensorRT-LLM ?

**Comment by the-david-oy** [comment, marked as answer]
Very happy to hear that, Matthieu! Thanks for sharing.

Triton will continue to work as the one solution for all of your AI model inferencing. We are working on a TensorRT-LLM backend that you can easily plug your models into.

**Reply by MatthieuToulemont** [reply]
Amazing !! Thank you for your answer
